### ISSUE TITLE
Changing the requestedStatus of a single service in the pipeline leads to Error

### ISSUE THREAD

**Issue by fr0gs** [big-data-europe/mu-swarm-admin-service#38]
Hello,

Following the same process to change a the pipeline's status but applied to a single service of the pipeline leads to an error:

```
swarm-admin_1  | INFO:aiohttp.access:172.20.0.5 - - [10/Jul/2017:14:06:12 +0000] "POST /update HTTP/1.1" 204 157 "-" "Java/1.8.0_45-internal"
swarm-admin_1  | INFO:aiohttp.access:172.20.0.5 - - [10/Jul/2017:14:06:12 +0000] "POST /update HTTP/1.1" 204 157 "-" "Java/1.8.0_45-internal"
swarm-admin_1  | INFO:aiohttp.access:172.20.0.5 - - [10/Jul/2017:14:06:12 +0000] "POST /update HTTP/1.1" 204 157 "-" "Java/1.8.0_45-internal"
swarm-admin_1  | INFO:aiohttp.access:172.20.0.5 - - [10/Jul/2017:14:06:12 +0000] "POST /update HTTP/1.1" 204 157 "-" "Java/1.8.0_45-internal"
swarm-admin_1  | INFO:muswarmadmin.services:Changing service 283065f5-cca4-48a3-a9fa-db3b679730bc status to <http://swarmui.semte.ch/vocabularies/core/Started>
swarm-admin_1  | INFO:muswarmadmin.main:Starting datanode ... 
swarm-admin_1  | INFO:muswarmadmin.main:[1A[2KStarting datanode ... [32mdone[0m[1BNo containers to start
```

The status is updated with this PATCH call:

```json
{
  "data": {
    "id": "a10ab8f6-d45e-4109-bf4c-94308a345602",
    "attributes": {
      "name": "hive-metastore-postgresql",
      "scaling": 0,
      "restart-requested": null
    },
    "relationships": {
      "pipeline-instance": {
        "data": {
          "type": "pipeline-instances",
          "id": "59636F5284D6CE0009000002"
        }
      },
      "status": {
        "data": {
          "type": "statuses",
          "id": "1BF9FED2-21C4-11E6-9AC5-A2FC35676F51"
        }
      },
      "requested-status": {
        "data": {
          "type": "statuses",
          "id": "31E7A0D0-7CA2-4C08-BFBC-E936F3FD9959"
        }
      }
    },
    "type": "services"
  }
}
```

The status in the virtuoso DB consistently ends up with:

```json
{
  "s": {
    "type": "uri",
    "value": "http:\/\/swarm-ui.big-data-europe.eu\/resources\/services\/283065f5-cca4-48a3-a9fa-db3b679730bc"
  },
  "p": {
    "type": "uri",
    "value": "http:\/\/swarmui.semte.ch\/vocabularies\/core\/status"
  },
  "o": {
    "type": "uri",
    "value": "http:\/\/swarmui.semte.ch\/vocabularies\/core\/Error"
  }
}
```

**Comment by fr0gs** [comment]
I know what the issue is. The service needs to be an existing and stopped container in order to be able to sepparately start it. 

I need to **Up** the pipeline, **Stop** it all and then **Start** the container I want to run standalone. Clicking on the Start button should create the container too.

**Comment by cecton** [comment]
Status Up has been implemented in 94b7bab26776f33ec56b82419ae2da085aa24886
